dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -875,6 +766,44 @@ void ExchangeReceiverBase<RPCContext>::connectionDone(
         finishAllMsgChannels();
 }
 
+template <typename RPCContext>
+void ExchangeReceiverBase<RPCContext>::connectionLocalDone(
```

Comment:
Seems most of the logical in `connectionLocalDone` is the same as `connectionDone`, can we reuse the code?

Reply:
> Seems most of the logical in `connectionLocalDone` is the same as `connectionDone`, can we reuse the code?

I have deleted `connectionLocalDone` and there is only `connectionDone` in the codes